src/Workspaces/CSharpTest/Formatting/FormattingTests_FunctionPointers.cs

```diff
@@ -31,19 +30,90 @@ unsafe class C
         }
 
         [Fact]
-        public async Task FormatFunctionPointerWithCallingConvention()
+        public async Task FormatFunctionPointerWithManagedCallingConvention()
         {
-            // TODO(https://github.com/dotnet/roslyn/issues/44312): add a space after the "int"s in the baseline and make this test still pass
             var content = @"
 unsafe class C
 {
-    delegate *cdecl < int,  int> functionPointer;
+    delegate *managed < int ,  int > functionPointer;
 }";
 
             var expected = @"
 unsafe class C
 {
-    delegate* cdecl<int, int> functionPointer;
+    delegate* managed<int, int> functionPointer;
+}";
+
+            await AssertFormatAsync(expected, content);
+        }
+
+        [Fact]
+        public async Task FormatFunctionPointerWithUnmanagedCallingConvention()
+        {
+            var content = @"
+unsafe class C
+{
+    delegate *unmanaged < int ,  int > functionPointer;
+}";
+
+            var expected = @"
+unsafe class C
+{
+    delegate* unmanaged<int, int> functionPointer;
+}";
+
+            await AssertFormatAsync(expected, content);
+        }
+
+        [Fact]
+        public async Task FormatFunctionPointerWithUnmanagedCallingConventionAndSpecifiers()
+        {
+            var content = @"
+unsafe class C
+{
+    delegate *unmanaged [ Cdecl ,  Thiscall ] < int ,  int > functionPointer;
+}";
+
+            var expected = @"
+unsafe class C
+{
+    delegate* unmanaged[Cdecl, Thiscall]<int, int> functionPointer;
+}";
+
+            await AssertFormatAsync(expected, content);
+        }
+
+        [Fact(Skip = "PROTOTYPE(func-ptr): Even though the rule is returning put a space, it's getting ignored, resulting in delegate*invalid <")]
+        public async Task FormatFunctionPointerWithUnrecognizedCallingConvention()
+        {
+            var content = @"
+unsafe class C
+{
+    delegate *invalid < int ,  int > functionPointer;
+}";
+
+            var expected = @"
+unsafe class C
+{
+    delegate* invalid<int, int> functionPointer;
+}";
+
+            await AssertFormatAsync(expected, content);
+        }
+
+        [Fact(Skip = "PROTOTYPE(func-ptr): Even though the rule is returning put a space, it's getting ignored, resulting in delegate*invalid [")]
```

Comment:
Ditto: as long s it's formatting the stuff that it does understand, I guess that's good.